src/main/java/cuchaz/enigma/utils/I18n.java

```diff
@@ -10,39 +10,34 @@
 import java.util.Map;
 
 import com.google.common.collect.Maps;
-import com.google.common.io.Resources;
 import com.google.gson.Gson;
 
 import cuchaz.enigma.config.Config;
 
-public class LangUtils {
+public class I18n {
 	public static final String DEFAULT_LANGUAGE = "en_us";
 	private static final Gson GSON = new Gson();
 	private static Map<String, String> translations = Maps.newHashMap();
 	private static Map<String, String> defaultTranslations = Maps.newHashMap();
 	private static Map<String, String> languageNames = Maps.newHashMap();
 	
 	static {
-		putInCache(Config.getInstance().language);
-		putInCache(DEFAULT_LANGUAGE);
+		translations = load(Config.getInstance().language);
+		defaultTranslations = load(DEFAULT_LANGUAGE);
 	}
 	
 	@SuppressWarnings("unchecked")
-	public static void putInCache(String language) {
-		try (InputStream inputStream = LangUtils.class.getResourceAsStream("/lang/" + language + ".json")) {
-			if (inputStream == null) {
-				return;
-			}
-			try (BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8))) {
-				if (language.equals(DEFAULT_LANGUAGE)) {
-					defaultTranslations = GSON.fromJson(reader, Map.class);
-				} else {
-					translations = GSON.fromJson(reader, Map.class);
+	public static Map<String, String> load(String language) {
+		try (InputStream inputStream = I18n.class.getResourceAsStream("/lang/" + language + ".json")) {
+			if (inputStream != null) {
+				try (BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8))) {
+					return GSON.fromJson(reader, Map.class);
 				}
 			}
 		} catch (IOException e) {
 			e.printStackTrace();
 		}
+		return null;
```

Comment:
my bad, i should ask to return `Collections.emptyMap()` so you don't get npe for calling `map.get`